src/test/resources/omop/snapshot-access-request.json

```diff
@@ -0,0 +1,5 @@
+{"sourceSnapshotId": "76aa1dfa-9475-4cee-b4b0-a954a27761be",
```

Comment:
placeholder, source snapshot id gets overridden in test

Reply:
Could you use all `0`s here so it's more apparent that it's a dummy value?